plenum/server/request_managers/write_request_manager.py

```diff
@@ -96,6 +107,16 @@ def dynamic_validation(self, request: Request):
         for handler in handlers:
             handler.dynamic_validation(request)
 
+    def _get_handlers_by_version(self, txn_type=None):
+        version = self.database_manager.state_version
+        version = self.config.INDY_NODE_VERSION_MATCHING.get(version, version)
```

Comment:
It would be great to either put a TODO comment here, or create a generic method `get_version` in `ITxnVersionController` which can take care on what is the current txn version